app/scripts/controllers/membership.js

```diff
@@ -27,7 +27,7 @@ angular
 
       // NOTE: these could all be moved out into a strings service.
       var messages = {
-        errorReason: _.template('Reason: "<%- httpErr %>"'),
+        errorReason: _.template('Reason: "<%= httpErr %>"'),
```

Comment:
At some point, we'll need a message catalog for translation. If we really want to pull the messages out of the controllers anyway, we should start using the format we'll eventually need for translation. But let's not worry about that for this PR.